catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -68,63 +78,134 @@ private RegistryPackageUtils() {
     }
 
     public static List<ServiceType> getServices(RegistryPackageType registryPackage) {
-        List<ServiceType> services = new ArrayList<>();
+        return getObjectsFromRegistryObjectList(registryPackage, ServiceType.class);
+    }
 
-        if (registryPackage == null) {
-            return services;
-        }
+    public static List<ServiceType> getServices(RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, ServiceType.class);
+    }
 
-        if (registryPackage.isSetRegistryObjectList()) {
-            services = getServices(registryPackage.getRegistryObjectList());
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, ExtrinsicObjectType.class);
+    }
+
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, ExtrinsicObjectType.class);
+    }
+
+    public static Map<String, Object> getServiceInfo(
```

Comment:
The registryPackageUtils just handles parsing and returning RegistryObjectTypes. If this is needed in a helper it should probably be added to the RegistryPackageWebConverter.